src/types/geojson.ts

```diff
@@ -1,4 +1,11 @@
-import GeoJSONType from '../enums/geojson';
+export enum GeoJSONType {
```

Comment:
An enum is not a type right. We do this because we moved it from a types file to a ts file right?

Reply:
It is a type, an enumerated type! 😄 The name somewhat threw me off, but it's the property for the `type` key, hence `Type`

My reasoning was that for the code base in its current size it's easier to keep it close to its related (GeoJSON) components, as I had to look for a bit while resolving related errors.